bindata/network/ovn-kubernetes/managed/004-config.yaml

```diff
@@ -0,0 +1,125 @@
+---
+# The ovnconfig config file. Used by both node and master processes.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovnkube-config
+  namespace: openshift-ovn-kubernetes
+data:
+  ovnkube.conf:   |-
+    [default]
+    mtu="{{.MTU}}"
+    {{- if .RoutableMTU }}
+    routable-mtu="{{.RoutableMTU}}"
+    {{- end }}
+    cluster-subnets="{{.OVN_cidr}}"
+    encap-port="{{.GenevePort}}"
+    enable-lflow-cache=true
+    lflow-cache-limit-kb=1048576
+
+    [kubernetes]
+    service-cidrs="{{.OVN_service_cidr}}"
+    ovn-config-namespace="openshift-ovn-kubernetes"
+    apiserver="{{.K8S_APISERVER}}"
+    host-network-namespace="openshift-host-network"
+{{- if .OVNHybridOverlayEnable }}
+    no-hostsubnet-nodes="kubernetes.io/os=windows"
+{{- end  }}
+    platform-type="{{.PlatformType}}"
+ 
+    [ovnkubernetesfeature]
+    enable-egress-ip=true
+    enable-egress-firewall=true
+
+    [gateway]
+    mode={{.OVN_GATEWAY_MODE}}
+    nodeport=true
+{{- if .OVNHybridOverlayEnable }}
+
+    [hybridoverlay]
+    enabled=true
+    {{- if .OVNHybridOverlayNetCIDR }}
+    cluster-subnets="{{.OVNHybridOverlayNetCIDR}}"
+    {{- end }}
+    {{- if .OVNHybridOverlayVXLANPort}}
+    hybrid-overlay-vxlan-port="{{.OVNHybridOverlayVXLANPort}}"
+    {{- end }}
+{{- end  }}
+{{- if .IsSNO }}
+
+    [masterha]
+    {{- /* 
+    Even in case of SNO there will be only one ovn-master, we set dedicated values for leader election 
+    durations in SNO, as disabling it can cause issues on scaling out SNO again. 
+    The whole discussion can be found at https://coreos.slack.com/archives/CDCP2LA9L/p1627402405090600. 
+    Recommended values at https://github.com/openshift/enhancements/blame/84e894ead7b188a1013556e0ba6973b8463995f1/CONVENTIONS.md#L183
+    */}}
+    election-lease-duration=137
+    election-renew-deadline=107
+    election-retry-period=26
+{{- end  }}
+
+
+{{- if .EnableHypershift }}
```

Comment:
Presumably we don't need this conditional, now that we split directories.

Reply:
Removed the conditional check. we shall always render it for managed deployment.